backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -34,6 +33,8 @@ export interface EmailTransactionalMiddleware {
 export const RATE_LIMIT_ERROR_MESSAGE =
   'Error 429: Too many requests, rate limit reached'
 
+export const TRANSACTIONAL_EMAIL_WINDOW = 1 // in seconds
```

Comment:
moving this to be a constant instead of an env var (no reason for us to need to change this quickly, probably will screw things up if we did...)